content/committees/steering/brand_use_by_projects.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Proper use of the Jakarta® Brand by Eclipse Projects"
+date: 2020-03-03T13:10:00-04:00
+---
+
+A critical aspect of managing a brand is avoiding confusion as to what the brand represents, what expectations should we expect to be met when something is labeled with that brand. It is important to clearly delineate what we mean when we label something “Jakarta”, that is, when we use “Jakarta” as a modifier/adjective to an object/noun (e.g., “Jakarta® Faces Specification”),  and managing how it is used in the project space is part of this. Careful use of the name will help the community better understand what is directly represented by the Jakarta brand and what is not.
+
+Specifications developed under the purview of the Jakarta EE Specification Committee must use the Jakarta brand in adjective form in their specification names (i.e., “Jakarta® [EE] X”). For example, the “Jakarta® EE Full Platform” or “Jakarta® Enterprise Beans”. All other uses of the Jakarta brand must use “Jakarta” as the object (i.e., “X for Jakarta® [EE]”). Including “EE” in either form is considered optional (and left to the discretion of the project team).
+
+By extension, only Eclipse open source specification projects that produce specifications under the purview of the Jakarta EE Working Group may use the brand in adjective form in specification project names. For example, the “Jakarta® Servlet” specification project produces the “Jakarta® Servlet” specification; and the “Jakarta® Stable APIs” specification project produces the “Jakarta® XML Registries” and “Jakarta® Deployment” specifications. All other Eclipse open source projects (and all uses outside of the scope of the Eclipse Foundation) must use the “for Jakarta®” form when including the term in the name is desired (e.g., “Eclipse Web Tools for Jakarta® EE”). 
+
+This has the following implications (this list may not be complete):
+
+* The Jakarta® EE Examples project will be renamed “Examples for Jakarta® EE”.
+
+Note that:
+
+* The trademarked term is “Jakarta” not “Jakarta EE”. The trademark annotations must be applied to the trademarked term (i.e., “Jakarta® EE”, not “Jakarta EE®”).
+* “Jakarta” is currently registered in the EU and the United States; so we must include “®” with the first use of the term in all documents and web pages.
```

Comment:
How does "first" vs. "most prominent" factor in here?  I'm not actually sure what trademark law says.